Gemfile

```diff
@@ -3,3 +3,7 @@
 source "https://rubygems.org"
 
 gemspec
+
+gem "aws-sdk-s3", "~> 1.67"
+gem "vcr"
+gem "webmock"
```

Comment:
@bronzdoc Where should I move these gems to?

Reply:
gemspec